kernel/src/lib.rs

```diff
@@ -63,8 +63,16 @@ pub mod actions;
 pub mod engine_data;
 pub mod error;
 pub mod expressions;
-pub(crate) mod predicates;
 pub mod table_features;
+pub mod scan;
+pub mod schema;
+pub mod snapshot;
+pub mod table;
+pub mod table_properties;
```

Comment:
only meaningful change is adding `pub mod table_properties`, other changes are shifting to colocate module declarations